webapp/src/components/modals/create_issue/create_issue.jsx

```diff
@@ -150,6 +145,13 @@ export default class CreateIssueModal extends PureComponent {
                     onChange={this.handleIssueTitleChange}
                 />
                 {issueTitleValidationError}
+
+                <GithubLabelSelector
+                    repo={this.state.repoValue}
+                    theme={theme}
+                    onChange={this.handleLabelsChange}
+                />
```

Comment:
Usually whenever a component is passing an `onChange` prop, there is always an accompanying `value` prop passed as well. This lets the parent hold the state so it's not duplicated in the child component.

Reply:
@mickmister See https://github.com/mattermost/mattermost-plugin-github/pull/343#discussion_r465818978